subsys/bluetooth/host/gatt.c

```diff
@@ -2940,11 +2940,13 @@ static int gatt_notify_multiple_verify_params(struct bt_conn *conn,
 			return -EINVAL;
 		}
 
-		/* Check if the characteristic is subscribed. */
-		if (!bt_gatt_is_subscribed(conn, params[i].attr,
-					   BT_GATT_CCC_NOTIFY)) {
-			LOG_WRN("Device is not subscribed to characteristic");
-			return -EINVAL;
+		if (IS_ENABLED(CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION)) {
+			/* Check if the characteristic is subscribed. */
+			if (!bt_gatt_is_subscribed(conn, params[i].attr,
+						   BT_GATT_CCC_NOTIFY)) {
+				LOG_WRN("Device is not subscribed to characteristic");
+				return -EINVAL;
+			}
```

Comment:
Suggest to reduce indent by 1 tab: 
```suggestion
		if (IS_ENABLED(CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION) && 
		    !bt_gatt_is_subscribed(conn, params[i].attr, BT_GATT_CCC_NOTIFY)) {
			LOG_WRN("Device is not subscribed to characteristic");
			return -EINVAL;
```